docs/_validate_sql_recovery_point.md

```diff
@@ -0,0 +1,22 @@
+# _validate_sql_recovery_point
+
+Checks whether a database exist on an SQL Instance and Host.
```

Comment:
Modify to `Checks whether the recovery point date and time are valid i.e. that database and log backups exist.`

Reply:
`Check whether the data and time provided is a valid recovery point for an MSSQL database`